cognite/neat/rules/transformers/_converters.py

```diff
@@ -61,7 +66,100 @@ def _transform(self, rules: T_VerifiedInRules) -> T_VerifiedOutRules:
         raise NotImplementedError()
 
 
-class ToCompliantEntityIds(ConversionTransformer[VerifiedRules, VerifiedRules]): ...
+class ToCompliantEntities(RulesTransformer[InformationInputRules, InformationInputRules]):  # type: ignore[misc]
+    """Converts input rules to rules with compliant entity IDs."""
```

Comment:
Is there some more documentation you can add here? Compliant with what?

Maybe just say which regex pattern you use with the properties

Reply:
Will add good point